css/scss/sections/_index.scss

```diff
@@ -0,0 +1,228 @@
+body {
+  background-color: #307367;
```

Comment:
You should get the colors from the color map, instead of hardcoding on each file separately.
That way if we need to change the color of the project we only change it in one place.
use:
set-color('accent-1')

Same for any other color you have on your file.

Reply:
fixed